assets/sass/components/key-metrics/_googlesitekit-key-metrics-site-purpose-change-modal.scss

```diff
@@ -20,6 +20,10 @@
 
 	.mdc-dialog.googlesitekit-dialog-confirm-site-purpose-change {
 
+		h2 {
+			font-size: 28px;
```

Comment:
Let's use the variable `$fs-headline-md` name instead of hardcoded number, or include `googlesitekit-heading-2` class on the heading element

Reply:
This has been fixed in the latest commit thanks @zutigrm